src/components/chart/_chart.scss

```diff
@@ -25,7 +25,7 @@
     }
 
     &__footnote-number {
-        display: flex;
+        display: flex !important;
```

Comment:
This important doesn't seem to be needed

Reply:
Actually it is in some cases. Here's a screen shot with it removed

![Screenshot 2025-05-16 at 08 31 34](https://github.com/user-attachments/assets/b834fdc4-970b-439a-a331-980f75697281)

I've added a comment to make it clearer.